gradle/dependencies.gradle

```diff
@@ -15,6 +15,8 @@ dependencies {
         exclude group: 'org.ow2.asm'
     }
 
+    implementation 'com.android.tools:r8:2.0.61'
```

Comment:
I think we should make this `compileOnly` and force the user to add their only r8 dependency so we don't have to maintain the compatibility. This should happened regardless of the extension implementation in the comment below.